app/controllers/hyrax/admin/permission_template_accesses_controller.rb

```diff
@@ -62,13 +57,18 @@ def after_destroy_error
           end
         end
 
-        def source_type
-          Hyrax::PermissionTemplate.find(@permission_template_access.permission_template_id).source_type
-        end
+        delegate :source_type, :source_id, to: :permission_template
 
         def remove_access!
-          Forms::PermissionTemplateForm.new(@permission_template_access.permission_template)
-                                       .remove_access!(@permission_template_access)
+          permisison_template_form.remove_access!(@permission_template_access)
+        end
+
+        def permisison_template_form
+          @permission_template_form ||= Forms::PermissionTemplateForm.new(permission_template)
+        end
+
+        def permission_template
+          @permission_template ||= @permission_template_access.permission_template
```

Comment:
Should this use the memoizing method instead of the ivar?

Reply:
I don't know what memorizing method you are referring to. Can you help me out?

Reply:
Uhhh, I don't know. I completely miffed on this. nvm!